server/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -1411,14 +1412,22 @@ private ApplicationBaseViewParams.Builder applicationBaseViewParamsBuilder(
       ApplicantRoutes applicantRoutes,
       CiviFormProfile profile) {
 
+    ImmutableList<ApplicantQuestion> ineligibleQuestions =
+        roApplicantProgramService.getIneligibleQuestions();
+    ImmutableList<String> formattedQuestions =
```

Comment:
Since we're doing this same mapping everywhere does it make sense to pass in the inelligible questions and map them to their text inside .calculate()?